src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
```suggestion
    public Color getRandomColor() {
```
as was mentioned in the checklist, avoid static methods